src/mainwindow.cpp

```diff
@@ -3671,6 +3671,13 @@ bool MainWindow::jumpToNoteSubFolder(int noteSubFolderId) {
     return false;
 }
 
+void MainWindow::selectNavigationItemAtPosition(int position)
```

Comment:
Would have been better if this could live in NavigationWidget, but I guess as things currently are, it will do.